core/api/service/chain/impl/chain_api_impl.cpp

```diff
@@ -0,0 +1,79 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "api/service/chain/impl/chain_api_impl.hpp"
+
+#include <boost/algorithm/hex.hpp>
+#include "common/hexutil.hpp"
+#include "common/visitor.hpp"
+
+namespace kagome::api {
+  using kagome::primitives::BlockHash;
```

Comment:
```suggestion
  using primitives::BlockHash;
```
or just write primitives::BlockHash everywhere, it is not that much letters:)